File: Baekjoon/No2750/Solution.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

int main()
{
    vector<int> ans;
    int num, temp;
    cin >> num;

    for (int i = 0; i < num; i++)
    {
        cin >> temp;
        ans.push_back(temp);
    }

    sort(ans.begin(), ans.end());

    for (int j = 0; j < num; j++)
        cout << ans[j] << endl;
}

File: Baekjoon/No2750/Solution.java
package Baekjoon.No2750;

import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int num = Integer.parseInt(br.readLine());
        int[] ans = new int[num];

        for (int i = 0; i < num; i++) {
            ans[i] = Integer.parseInt(br.readLine());
        }

        Arrays.sort(ans);

        for (int j = 0; j < num; j++) {
            System.out.print(ans[j]);
            System.out.print("\n");
        }
    }
}